### ISSUE TITLE
weave reset does not remove net-plugin v2 weavedata.db

### ISSUE THREAD

**Issue by dtshepherd** [weaveworks/weave#3103]
## What you expected to happen?

After running `weave reset --force`, `/var/lib/weave/weavedata.db` would be removed.

## What happened?

`/var/lib/weave/weavedata.db` still exists on the host.

## How to reproduce it?

1. Install net-plugin v2.
2. Disable and remove the plugin.
3. Run `weave reset --force`

## Anything else we need to know?

This is a similar issue that was fixed with PR #2610.

## Versions:

```
$ weave version
weave script 2.0.4
weave 2.0.4
$ docker version
Client:
 Version:      17.06.0-ce
 API version:  1.30
 Go version:   go1.8.3
 Git commit:   02c1d87
 Built:        Fri Jun 23 21:23:31 2017
 OS/Arch:      linux/amd64

Server:
 Version:      17.06.0-ce
 API version:  1.30 (minimum version 1.12)
 Go version:   go1.8.3
 Git commit:   02c1d87
 Built:        Fri Jun 23 21:19:04 2017
 OS/Arch:      linux/amd64
 Experimental: false
$ uname -a
Linux NUC 4.4.0-87-generic #110-Ubuntu SMP Tue Jul 18 12:55:35 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```

## Logs:

NA

## Network:

NA

**Comment by brb** [comment]
Thanks for the report.

**Comment by dtshepherd** [comment]
FYI, I think I have a fix for this.  Similar to have removal is handled for `weave-kube`.

**Comment by brb** [comment]
Fixed by #3114
